packages/react/src/lib/accessibility/types.ts

```diff
@@ -30,6 +30,7 @@ export type AriaWidgetRole =
   | 'tooltip'
   | 'treeitem'
   | 'switch'
+  | 'none'
```

Comment:
is role `none` supported by all screen readers?
from what I found in one discussion
```
JAWS supports role = "none", but NVDA does not.
Both JAWS and NVDA support role = "presentation".
Only JAWS works with role = "none presentation", even though its usage is flawed.
```

But it might be outdated, just making sure that it works

Reply:
I tested and it works. Thanks for pointing it out!